src/TibiaWorldsWorldV3_test.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWintera(t *testing.T) {
+	data, err := ioutil.ReadFile("../testdata/worlds/world/Wintera.html")
+	if err != nil {
+		t.Errorf("File reading error: %s", err)
+		return
+	}
+
+	worldJson := TibiaWorldsWorldV3Impl("Wintera", string(data))
+	assert := assert.New(t)
+
+	world := worldJson.Worlds.World
+
+	assert.Equal("Wintera", world.Name)
+	assert.Equal("online", world.Status)
+	assert.Equal(281, world.PlayersOnline)
+	assert.Equal(1023, world.RecordPlayers)
+	assert.Equal("2020-05-04T01:25:30Z", world.RecordDate)
+	assert.Equal("04/18", world.CreationDate)
+	assert.Equal("North America", world.Location)
+	assert.Equal("Open PvP", world.PvpType)
+	assert.Equal(false, world.PremiumOnly)
+	assert.Empty(world.TransferType)
+	assert.Equal(4, len(world.WorldsQuestTitles))
+	assert.Equal("A Piece of Cake", world.WorldsQuestTitles[0])
+	assert.Equal("Rise of Devovorga", world.WorldsQuestTitles[1])
+	assert.Equal("Bewitched", world.WorldsQuestTitles[2])
+	assert.Equal("The Colours of Magic", world.WorldsQuestTitles[3])
+	assert.Equal(true, world.BattleyeProtected)
+	assert.Equal("April 19, 2018", world.BattleyeDate)
```

Comment:
```suggestion
	assert.Equal("2018-04-19", world.BattleyeDate)
```

after rebase, the value of the return should be formatted by TibiadataDateV3